hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateNftsHandler.java

```diff
@@ -85,7 +84,8 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         final var txn = context.body();
         final var op = txn.tokenUpdateNftsOrThrow();
         final var tokenStore = context.createStore(ReadableTokenStore.class);
-        final var token = getIfUsable(op.tokenOrElse(TokenID.DEFAULT), tokenStore);
+        final var token = tokenStore.get(op.tokenOrElse(TokenID.DEFAULT));
```

Comment:
Line 78 also should be `validateTruePrecheck`

Reply:
Updated this.

Reply:
I didn't see it was updated
```
validateTrue(!op.serialNumbers().isEmpty(), MISSING_SERIAL_NUMBERS);
```